.gitignore

```diff
@@ -22,3 +22,6 @@ elastic-package
 
 # Build directory
 /build
+
+# stack dump
+elastic-stack-dump/
```

Comment:
Do you need this? We are already excluding `/build` and from what I can tell `elastic-stack-dump` lives under `build/`.